news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md

```diff
@@ -0,0 +1,57 @@
+
+In this article, we will try to answer all your questions about this subject. So let’s get started!
+
+## What is a built-in module exactly?
+
+A built-in module is a module that was developed by PrestaShop and that is available in your back office either in the Module Manager or in the Module Catalog. 
+
+We are not talking about "native modules" here, as the definition of a native module is a bit more complicated. Indeed, some native modules are present in the installation zip, some are pre-installed and some others are not. So, to make it easier, we have chosen to talk about built-in modules in this article.
+
+## Where can I find the list of all built-in modules?
+
+There was a time where it was difficult to find the list of all built-in modules. You could have a part of it directly in the back office but half of the information was missing and most of it has not been updated (or worse, inherited from the 1.6 version). In the end, it was not helpful at all.
+
+Dear PrestaShop users, those days are over! 
+
+Our core product team took some time to test all modules, screenshot them, accurately categorize them, write explicit and complete product information, translate it into eight languages, create special Addons pages for them and gather it all in a dedicated brochure available  [here](https://www.prestashop.com/fr/ressources).
+
+This brochure, available in no less than 8 languages, will give you a complete overview of the natively available features that will help you empower your business!
+
+Spoiler alert: It will soon be possible to click directly on the brochure to access its page on the PrestaShop Addons marketplace! A Russian version of the brochure will also be released soon.
+
+## The current logos look a bit outdated, are you going to update them?
+
+![Tristan’s quote](/assets/images/2021/01/build-article-modules-tristan-january21.png)
+
+It was time for a new look! Our Design team did a great job at modernizing and freshening the logos. Icons are pretty much the same but their colors fit better with the PrestaShop identity, the user experience will be more consistent. 
+
+These logos will be available in the back office during the first semester of 2021 (We have to wait for the QA to validate the pull requests, and for the releases to be done… Please just hold a little longer, you will soon be able to see it in your back office. To help you wait here's a little preview:
+
+![New logos](/assets/images/2021/01/build-article-modules-logos-january21.png)
+
+You can also have a look at the [GitHub issue](https://github.com/PrestaShop/PrestaShop/issues/15858).
+
+## What if I want to know more about a module in particular?
+
+To know more about a specific module (whether it is a built-in module or not) you should visit the [PrestaShop Official Addons Marketplace](https://addons.prestashop.com/en/).
+
+Our product managers have redesigned the built-in module pages on the marketplace to make them clearer and more understandable. 
```

Comment:
could you please add a link to this page?

Reply:
Well, this is not just one page. There are as many pages as there are built-in modules. All these pages are available on the Addons Marketplace.
I plan to add the links to each page on the brochures, so access to those pages become easier for the merchants. 😊

Reply:
ok, then, maybe you can tell something about that in the article.
thanks ;-)